naming/src/main/java/com/alibaba/nacos/naming/core/InstanceOperatorClientImpl.java

```diff
@@ -97,6 +98,12 @@ public InstanceOperatorClientImpl(ClientManagerDelegate clientManager,
      */
     @Override
     public void registerInstance(String namespaceId, String serviceName, Instance instance) {
+        try {
```

Comment:
The interface of `registerInstance` can throw NacosException. you can see `InstanceOperator`.

So no need to catch NacosException and translate to NacosRuntimeException